vllm/model_executor/layers/quantization/tpu_int8.py

```diff
@@ -0,0 +1,114 @@
+from typing import Any, Dict, List, Optional, Tuple
+
+import torch
+from torch.nn import Module
+from torch.nn.parameter import Parameter
+
+from vllm.model_executor.layers.linear import (LinearBase, LinearMethodBase)
+from vllm.model_executor.layers.quantization.base_config import (
+    QuantizationConfig)
+from vllm.model_executor.utils import set_weight_attrs
+
+import torch_xla.experimental.xla_quantized_matmul
```

Comment:
Please move this import into your create_weights function or somewhere it will be lazy-imported and wrap it in a try-except. Here is deepspeedfp as an example: https://github.com/vllm-project/vllm/blob/460c1884e3cb781730f85cb5591a85d5864bdac8/vllm/model_executor/layers/quantization/deepspeedfp.py#L144-L153

Reply:
Hi @mgoin, thank you for pointing it out! Updated